src/main/resources/templates/db_scripts/publication-period/migratePublicationPeriod.sql

```diff
@@ -0,0 +1,127 @@
+CREATE OR REPLACE FUNCTION ${myuniversity}_${mymodule}.migrate_publication_period(
+    jsonb_data jsonb
+) RETURNS jsonb AS $$
+DECLARE
+    pub_period jsonb;
+    start_date text;
+    end_date text;
+    date_type_id text;
+    dates jsonb := '{}'::jsonb;
+BEGIN
+    pub_period := jsonb_data -> 'publicationPeriod';
+    start_date := pub_period ->> 'start';
+    end_date := pub_period ->> 'end';
```

Comment:
We should truncate these values to null if it's just a blank string.

Reply:
done